org.openjdk.jmc.console.ext.agent/src/main/java/org/openjdk/jmc/console/ext/agent/manager/wizards/EventEditingWizardConfigPage.java

```diff
@@ -66,13 +113,100 @@ public void createControl(Composite parent) {
+		createSeparator(container).setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, false));
+		createMetaInfoContainer(container).setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, false));
+
+		bindListeners();
+		populateUi();
 
 		sc.setExpandHorizontal(true);
 		sc.setExpandVertical(true);
 		sc.setMinSize(container.computeSize(SWT.DEFAULT, SWT.DEFAULT));
 		setControl(sc);
 	}
+
+	private Composite createConfigContainer(Composite parent) {
+		Composite container = new Composite(parent, SWT.NONE);
+		int cols = 5;
+		GridLayout layout = new GridLayout(cols, false);
+		layout.horizontalSpacing = 8;
+		container.setLayout(layout);
+
+		idText = createTextInput(container, cols, LABEL_ID, MESSAGE_EVENT_ID);
+		nameText = createTextInput(container, cols, LABEL_NAME, MESSAGE_NAME_OF_THE_EVENT);
+		descriptionText = createMultiTextInput(container, cols, LABEL_DESCRIPTION,
+				MESSAGE_OPTIONAL_DESCRIPTION_OF_THIS_EVENT);
+
+		return container;
+	}
+
+	private Composite createInstrumentationTargetContainer(Composite parent) {
+		Composite container = new Composite(parent, SWT.NONE);
+		int cols = 8;
+		GridLayout layout = new GridLayout(cols, false);
+		layout.horizontalSpacing = 8;
+		container.setLayout(layout);
+
+		classText = createTextInput(container, cols, LABEL_CLASS, MESSAGE_FULLY_QUALIFIED_CLASS_NAME);
+		Text[] receivers = createMultiInputTextInput(container, cols, LABEL_METHOD,
+				new String[] {MESSAGE_METHOD_NAME, MESSAGE_METHOD_DESCRIPTOR});
+
+		methodNameText = receivers[0];
+		methodDescriptorText = receivers[1];
+
+		return container;
+	}
+
+	private Composite createMetaInfoContainer(Composite parent) {
+		Composite container = new Composite(parent, SWT.NONE);
+		int cols = 8;
+		GridLayout layout = new GridLayout(cols, false);
+		layout.horizontalSpacing = 8;
+		container.setLayout(layout);
+
+		pathText = createTextInput(container, cols, LABEL_PATH, MESSAGE_PATH_TO_EVENT);
+		locationCombo = createComboInput(container, cols - 2, LABEL_LOCATION,
+				Stream.of(Location.values()).map(Location::toString).toArray(String[]::new));
+		locationClearBtn = createButton(container, LABEL_CLEAR);
+		locationClearBtn.setLayoutData(new GridData(SWT.FILL, SWT.CENTER, false, false, 2, 0));
+
+		recordExceptionsBtn = createCheckboxInput(container, cols, LABEL_RECORD_EXCEPTIONS);
+		recordStackTraceBtn = createCheckboxInput(container, cols, LABEL_RECORD_STACK_TRACE);
+
+		return container;
+	}
+
+	private void bindListeners() {
+		idText.addModifyListener(e -> event.setId(idText.getText()));
+		nameText.addModifyListener(e -> event.setName(nameText.getText()));
+		descriptionText.addModifyListener(e -> event.setDescription(descriptionText.getText()));
+		methodNameText.addModifyListener(e -> event.setMethodName(methodNameText.getText()));
+		methodDescriptorText.addModifyListener(e -> event.setMethodDescriptor(methodDescriptorText.getText()));
+		pathText.addModifyListener(e -> event.setMethodName(pathText.getText()));
```

Comment:
this should be `pathText.addModifyListener(e -> event.setPath(pathText.getText()));` 
instead of `event.setMethodName(..)`

Reply:
Oops... Thank you for catching this!